app/Controller/PagesController.php

```diff
@@ -72,5 +122,59 @@ public function display() {
 			}
 			throw new NotFoundException();
 		}
-	}
+  }
+
+  private $顧客名;
+
+  function set顧客名($v)
+  {
+    $this->顧客名 = $v;
+    return $this;
+  }
+
+  function get顧客名()
+  {
+    return $this->顧客名;
+  }
+}
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
+        if ($a == $b)  {
+            if ($a1 == $b1) {
+                fiddle();
+            } else if ($a2 == $b2) {
```

Comment:
__[code_sniffer]__
* Usage of ELSE IF is discouraged; use ELSEIF instead
* Usage of ELSE IF not allowed; use ELSEIF instead

<!-- Comment by [SideCI](https://sideci.com) -->